lib/SILGen/SILGenStmt.cpp

```diff
@@ -318,6 +318,14 @@ void StmtEmitter::visitBraceStmt(BraceStmt *S) {
       } else {
         diagnose(getASTContext(), ESD.getStartLoc(),
                  diag::unreachable_code);
+				if (!S->getElements().empty()) {
+					for (auto *arg : SGF.getFunction().getArguments()) {
+						if (arg->getType().getASTType()->isStructurallyUninhabited()) {
+							diagnose(getASTContext(), S->getStartLoc(), diag::unreachable_code_uninhabited_param_note, arg->getDecl()->getBaseName().userFacingName());
+							break;
+						}
+					}
+				}
```

Comment:
Looks like tabs snuck in instead of spaces here!

Reply:
I've fixed it by running `git-clang-format`.